Minio.Tests/NegativeTest.cs

```diff
@@ -63,10 +63,17 @@ public async Task TestInvalidObjectNameError()
 
             try
             {
+                const int tryCount = 5;
                 await minio.MakeBucketAsync(new MakeBucketArgs(bucketName));
 
                 var ex = await Assert.ThrowsExceptionAsync<InvalidObjectNameException>(
                     () => minio.StatObjectAsync(bucketName, badName));
+                for(int i=0; i < tryCount && !(ex.Response.Code.Equals("InvalidObjectName")); ++i)
```

Comment:
we should check for http status codes to be in 500s for this retry

Reply:
Done.